cmd/vicadmin/server.go

```diff
@@ -47,11 +52,21 @@ const (
 	formatZip
 )
 
+const (
+	sessionExpiration      = time.Hour * 24
+	sessionCookieKey       = "sessiondata"
+	sessionCreationTimeKey = "created"
+	sessionUsernameKey     = "username"
+	loginPagePath          = "/authentication"
+	authFailure            = loginPagePath + "?unauthorized"
+	genericErrorMessage    = "Internal Server Error; see /var/log/vic/vicadmin.log for details" // for http errors that shouldn't be displayed in the browser to the user
```

Comment:
Reference to /var/log/vic/vicadmin.log does not really mean anything to customers or api callers. Instead a reference to the vicadmin portal might be better.